pippo-server-parent/pippo-jetty/src/main/java/ro/pippo/jetty/JettyServer.java

```diff
@@ -58,46 +55,8 @@
 
     private Server server;
 
-    private final ExecutorService executor;
-    private final CountDownLatch  startLatch;
-
-    public JettyServer() {
```

Comment:
I think that we need this constructor with `super()`. See https://github.com/decebals/pippo/blob/master/pippo-core/src/main/java/ro/pippo/core/AbstractWebServer.java#L36.